schemas/constructs/v1beta1/designs.json

```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Design Schema",
+  "description": "Designs are your primary tool for collaborative authorship of your infrastructure, workflow, and processes.",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the design; a descriptive, but concise title for the design document."
+    },
+    "schemaVersion": {
+      "$ref": "../v1alpha2/core.json#/definitions/versionString",
+      "description": "Specifies the version of the schema to which the design  conforms."
+    },
+    "version": {
+      "$ref": "../v1alpha2/core.json#/definitions/semverString",
+      "description": "Revision of the design as expressed by an auto-incremented, SemVer-compliant version number. May be manually set by a user or third-party system, but will always be required to be of version number higher than the previously defined version number.",
+      "minLength": 1,
+      "maxLength": 50
+    },
+    "components": {
+      "description": "Map of component IDs to their corresponding component declarations",
+      "additionalProperties": {
+        "type": "string",
+        "format": "uuid"
+      },
+      "type": "object",
+      "properties": {
+        "schemaVersion": {
+          "$ref": "./component.json#/properties/schemaVersion"
+        },
+        "version": {
+          "$ref": "./component.json#/properties/version"
+        },
+        "name": {
+          "$ref": "../v1alpha2/core.json#/definitions/inputString"
+        },
+        "model": {
+          "$ref": "./component.json#/properties/model"
+        },
+        "metadata": {

```

Comment:
It'd be nice to move metadata above model and not in-between model and component.